src/hocon_schema.erl

```diff
@@ -298,6 +303,14 @@ maybe_add_desc(Mod, Name, Meta) ->
             Meta
     end.
 
+inherit_hidden(Parent, Fields) when is_map(Fields) ->
```

Comment:
do we still need this?

Reply:
I thought this was needed, but later I added the `reject_hidden` filter it seems no longer necessary.